packages/web-components/fast-foundation/src/avatar/avatar.ts

```diff
@@ -0,0 +1,98 @@
+import { attr, FASTElement } from "@microsoft/fast-element";
+
+type AvatarShape = "circle" | "square";
+
+/**
+ * An Avatar Custom HTML Element
+ *
+ * @public
+ */
+export class Avatar extends FASTElement {
+    /**
+     * Indicates the Avatar should have a name.
+     *
+     * @public
+     * @remarks
+     * HTML Attribute: name
+     */
+    @attr public name: string;
+    protected nameChanged(): void {
+        const result = this.name?.match(/\b\w/g) || [];
+        this.initials = ((result.shift() || "") + (result.pop() || "")).toUpperCase();
```

Comment:
What happens if the initials derived here are different than what is set in the initials attribute, if they set both but the initials are something different than in the name they'll be overwritten? Is that acceptable? It might be I'm just asking the question.

Reply:
Added a check to see if an initial is given. If there is no initial given it defaults to the initialize name.